common/autoware_overlay_rviz_plugin/autoware_minimap_overlay_rviz_plugin/README.md

```diff
@@ -0,0 +1,93 @@
+# autoware_minimap_overlay_rviz_plugin
+
+Plugin for displaying a minimap overlay in RViz2 with vehicle position, goal pose, and route information.
+
+Based on the [jsk_visualization](https://github.com/jsk-ros-pkg/jsk_visualization) package, under the 3-Clause BSD license.
+
+## Purpose
+
+This plugin provides a visual representation of the vehicle's position, the goal pose, and the planned route on a minimap overlay within RViz2. It is designed to enhance situational awareness by displaying relevant navigation information.
+
+## Inputs / Outputs
+
+### Input
+
+| Name                                             | Type                                             | Description                           |
+| ------------------------------------------------ | ------------------------------------------------ | ------------------------------------- |
+| `/api/vehicle/kinematics`                        | `autoware_adapi_v1_msgs::msg::VehicleKinematics` | The topic is the vehicle's kinematics |
+| `/planning/mission_planning/echo_back_goal_pose` | `geometry_msgs::msg::PoseStamped`                | The topic is the goal pose            |
+| `/api/routing/state`                             | `autoware_adapi_v1_msgs::msg::RouteState`        | The topic is the routing state        |
+| `/planning/scenario_planning/trajectory`         | `autoware_planning_msgs::msg::Trajectory`        | The topic is the planned trajectory   |
```

Comment:
Do we want to visualize trajectory? The trajectory only contains the local path, and not the full path to the goal. If we want to visualize the whole route, then we might want to subscribe to a different topic...

Reply:
Yes, I think it gives a nice visual to see the small version of the path slowly change as the vehicle moves along instead of having the full route drawn.